src/Network/Wai/Handler/Hal.hs

```diff
@@ -216,6 +216,15 @@ runWithContext opts app ctx req = liftIO $ do
 -- | Convert the request sent to a Lambda serving an API Gateway proxy
 -- integration into a WAI request.
 --
+-- When processing multiple values for the same
+-- [HTTP header|query parameter], API Gateway will put the final such
+-- value into ['HalRequest.headers'|'HalRequest.queryStringParameters']
+-- and all such values into
+-- ['HalRequest.multiValueHeaders'|'HalRequest.multiValueQueryStringParameters'].
```

Comment:
This was meant to be placeholder text for two separate comment blocks. Does Haddock render it cleanly or make a mess?